versioned_docs/version-2.0/final-project/intro.mdx

```diff
@@ -39,6 +39,6 @@ Pairing or working in a group is highly recommended. You can also work alone, bu
      - Show the outcome
      - Talk about the learnings you've made & possible extensions of the project
 
-For the final project you have the entire day. Note that this is not much time. Therefore, be well-structured
+Note for this final project you don't have much time. Therefore, be well-structured
```

Comment:
The new schedule has half a day instead of the entire day. However, I suggest it's better to leave off the exact time so this won't have to change as the schedule does.